test/test-fs.c

```diff
@@ -2313,6 +2383,65 @@ TEST_IMPL(fs_futime) {
 #endif
 }
 
+TEST_IMPL(fs_futime_ex) {
+  utime_check_t checkme;
+  const char* path = "test_file";
+  double btime;
+  double atime;
+  double mtime;
+  uv_os_fd_t file;
+  uv_fs_t req;
+  int r;
+
+  /* Setup. */
+  loop = uv_default_loop();
+  unlink(path);
+  r = uv_fs_open(NULL, &req, path, O_RDWR | O_CREAT, S_IWUSR | S_IRUSR, NULL);
+  ASSERT(r >= 0);
+  ASSERT(req.result >= 0);
+  uv_fs_req_cleanup(&req);
+  close(r);
+
+  btime = atime = mtime = 400497753; /* 1982-09-10 11:22:33 */
+
+  r = uv_fs_open(NULL, &req, path, O_RDWR, 0, NULL);
+  ASSERT(r >= 0);
+  ASSERT(req.result >= 0);
+  file = req.result; /* FIXME probably not how it's supposed to be used */
```

Comment:
FWIW, in test-fs.c, we do this `fd = uv_get_osfhandle(open_req1.result);`

Reply:
Missed this one.

Reply:
Yeah, my bad.